frontend/src/components/Avatar/Avatar.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+import styles from './avatar.module.css';
+import avatar from './assets/avatar.png';
+
+const Avatar = ({ source, size }) => {
+  return (
+    <img
+      src={source}
+      alt="avatar"
+      className={`${styles.avatar} ${styles[size]}`}
+    />
+  );
+};
+
+Avatar.propTypes = {
+  source: PropTypes.string,
+  size: PropTypes.string,
```

Comment:
Let's limit available options for `size` prop

so we can only pass "small", "medium" or "large"